genon/preprocessor/facade/parser_processor.py

```
@@ -0,0 +1,1520 @@
+            loader = AudioLoader(
+                file_path=file_path,
+                req_url=self._whisper_url,
+                req_data=self._whisper_req_data,
+                chunk_sec=self._whisper_chunk_sec,
+                tmp_path=tmp_path,
+            )
+            audio_chunks = loader.split_file_as_chunks()
+            return loader.transcribe_audio(audio_chunks)
+        finally:
+            try:
+                subprocess.run(["rm", "-r", tmp_path], check=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)